src/scenes/App/components/TermsAndConditions/index.jsx

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import { withStyles } from 'material-ui/styles';
+import Paper from 'material-ui/Paper';
+import Typography from 'material-ui/Typography';
+import { FormattedMessage } from 'react-intl';
+import Grid from 'material-ui/Grid';
+import { FormGroup, FormControlLabel } from 'material-ui/Form';
+import Checkbox from 'material-ui/Checkbox';
+import Button from 'material-ui/Button';
+
+import AppConfig from '../../../../config/app';
+import styles from './styles';
+
+const TNC_ACCEPTED = 'tncAccepted';
+
+@withStyles(styles, { withTheme: true })
+export default class TermsAndConditions extends React.PureComponent {
```

Comment:
```
import React, { Component } from 'react';
export default class TermsAndConditions extends Component {
  ...
}
```

Reply:
Let's keep things consistent, also, PureComponent is not necessary here

Reply:
sounds good. i need to read up the diff between pure component and component.